src/Robo/Commands/Drupal/DrupalCommand.php

```diff
@@ -20,14 +21,16 @@ class DrupalCommand extends BltTasks {
    */
   public function install() {
 
+    $account_name = 'super-user-' . RandomString::string(10);
```

Comment:
Core, at least D7, makes this pretty easy -- https://www.drupal.org/user/autocomplete/super-